vendor/bat-native-ads/src/bat/ads/internal/client/client.cc

```diff
@@ -48,7 +48,7 @@ FilteredCategoryList::iterator FindFilteredCategory(
   DCHECK(filtered_categories);
 
   return std::find_if(filtered_categories->begin(), filtered_categories->end(),
-                      [&name](const FilteredCategory& category) {
+                      [&name](const FilteredCategoryInfo& category) {
```

Comment:
nit: would it make sense to refactor `category` to `segment` while we're touching it?

Reply:
Unfortunately not as quite a big change and this was planned when we move history to a database. Your thoughts?